oidc/oidc.go

```diff
@@ -450,6 +452,8 @@ func (i *IDToken) VerifyAccessToken(accessToken string) error {
 		h = sha512.New384()
 	case RS512, ES512, PS512:
 		h = sha512.New()
+	case EdDSA:
+		return errUnsupportedAlgorithm
```

Comment:
wouldn't this be sha512.New()?

Reply:
I believe that verifying EdDSA key with ed25519 protocol is a bit more complicated than the pattern that exists here [short explanation](https://security.stackexchange.com/questions/233099/validating-an-ed25519-public-key#:~:text=Yes.,see%20if%20the%20equation%20checks.)

Reply:
This is only for the at_hash value, it's not doing full verification :)